daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md

```diff
@@ -140,5 +140,22 @@ Console.WriteLine("Got a secret value, I'm not going to be print it, it's a secr
 
 - For a full guide on secrets visit [How-To: Retrieve secrets]({{< ref howto-secrets.md >}}).
 
+### Get Configuration Keys
+```csharp
+var client = new DaprClientBuilder().Build();
+
+// Retrieve a specific set of keys.
+var specificItems = await client.GetConfiguration("configstore", new List<string>() { "key1", "key2" });
```

Comment:
LGTM in terms of a .NET/C# API that closely maps to the REST'ful API.  Approve.

This is the alpha release.  That said in the future I'd like to review underlying design.  I'm curious why this is based on index arrays.  This feels like a dictionary scenario more like Config system or Env Vars.

Reply:
Yeah I think it'll be good to discuss how we can use this API as we start to see people adopting it in its alpha stage.